src/index.ts

```diff
@@ -46,6 +47,10 @@ app.use((req, res, next) => {
 const provider = getProvider();
 const { ovalConfigs } = env;
 
+// Initialize unlocker wallets for each Oval instance.
+const keyManager = PrivateKeyManager.getInstance();
+keyManager.initialize(ovalConfigs);
```

Comment:
you might just want to use `.catch` for errors and log and/or optionally exit process if this fails